app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    temp_arr = command.split()
+    if len(temp_arr) == 3 and temp_arr[1] != temp_arr[2]:
```

Comment:
You should make sure that your command starts with `cp` and perform unpacking to be more explicit. Prefer not using indexing when you have expectations that certain information is placed on certain position:
```suggestion
    command_keyword, initial_filename, copied_filename, _* = command.split()
    if command_keyword == "cp" and initial_filename != copied_filename:
```